UNITTESTS/stubs/ATHandler_stub.cpp

```diff
@@ -405,3 +406,8 @@ void ATHandler::set_debug_list(bool debug_on)
     ATHandler_stub::debug_on = debug_on;
 }
 
+void ATHandler::set_send_delay(uint16_t send_delay)
+{
+    ATHandler_stub::send_delay = send_delay;
```

Comment:
Could be left empty as ATHandler_stub::send_delay is not used. Also then could remove ATHandler_stub::send_delay.